### DISCUSSION TITLE
[ERROR]: gitstatus failed to initialize.

### DISCUSSION THREAD

**Discussion by ehackett00** [ohmyzsh/ohmyzsh#9852] [Help] [answered]
Hello,

I am receiving the following error from Oh-My-zsh:

     [ERROR]: gitstatus failed to initialize.

I do not have any idea why this is happening.  I do not know if this is something I need to be worried about.  Can you help?

I did manage to get a debug log:

[ERROR]: gitstatus failed to initialize.

  Zsh log (/private/var/folders/mn/zftkdvk150v98011k4dmb_zr0000gn/T/gitstatus.POWERLEVEL9K.501.67356.1619095118.2.xtrace.log):

    +(anon):7> setopt monitor
    +(anon):9> ((  ! _GITSTATUS_STATE_POWERLEVEL9K  ))
    +(anon):10> [[ -r /proc/version ]]
    +(anon):13> print -rn
    +(anon):14> zsystem flock -f lock_fd /private/var/folders/mn/zftkdvk150v98011k4dmb_zr0000gn/T/gitstatus.POWERLEVEL9K.501.67356.1619095118.2.lock
    +(anon):15> [[ 14 == <1-> ]]
    +(anon):18> typeset -gi '_GITSTATUS_LOCK_FD_POWERLEVEL9K=lock_fd'
    +(anon):20> [[ darwin20.0 == cygwin* ]]
    +(anon):41> sysopen -r -o cloexec -u resp_fd /dev/fd/13
    +(anon):44> typeset -gi 'GITSTATUS_DAEMON_PID_POWERLEVEL9K=67408'
    +(anon):46> [[ 16 == <1-> ]]
    +(anon):47> typeset -gi '_GITSTATUS_RESP_FD_POWERLEVEL9K=resp_fd'
    +(anon):48> typeset -gi '_GITSTATUS_STATE_POWERLEVEL9K=1'
    +(anon):51> ((  ! async  ))
    +(anon):52> ((  _GITSTATUS_CLIENT_PID_POWERLEVEL9K == sysparams[pid]  ))
    +(anon):54> local pgid
    +(anon):55> ((  0 < 20  ))
    +(anon):56> [[ -t 16 ]]
    +(anon):57> sysread -s 20 -t 5.0000000000 -i 16 'pgid[$#pgid+1]'
    +(anon):41> _gitstatus_daemon_p9k_
    +_gitstatus_daemon_p9k_:1> local -i pipe_fd
    +(anon):55> ((  20 < 20  ))
    +(anon):59> [[ '               67408' == \ #<1-> ]]
    +(anon):60> typeset -gi 'GITSTATUS_DAEMON_PID_POWERLEVEL9K=pgid'
    +(anon):62> sysopen -w -o cloexec -u req_fd -- /private/var/folders/mn/zftkdvk150v98011k4dmb_zr0000gn/T/gitstatus.POWERLEVEL9K.501.67356.1619095118.2.fifo
    +(anon):63> [[ 15 == <1-> ]]
    +(anon):64> typeset -gi '_GITSTATUS_REQ_FD_POWERLEVEL9K=req_fd'
    +(anon):66> print -nru 15 -- $'}hello\C-_\C-^'
    +(anon):67> local expected=$'}hello\C-_0\C-^' actual
    +(anon):68> ((  1  ))
    +(anon):68> [[ ! -t 1 ]]
    +(anon):71> local -F deadline=1
    +(anon):73> true
    +(anon):74> [[ -t 16 ]]
    +(anon):75> sysread -s 1 -t 5.0000000000 -i 16 actual
    +(anon):75> return

  Daemon log (/private/var/folders/mn/zftkdvk150v98011k4dmb_zr0000gn/T/gitstatus.POWERLEVEL9K.501.67356.1619095118.2.daemon.log):

    +_gitstatus_daemon_p9k_:3> local pgid=67408
    +_gitstatus_daemon_p9k_:4> [[ 67408 == <1-> ]]
    +_gitstatus_daemon_p9k_:5> cd -q /
    +_gitstatus_daemon_p9k_:90> ((  lock_fd == -1  ))
    +_gitstatus_daemon_p9k_:9> trap '' PIPE
    +_gitstatus_daemon_p9k_:11> local uname_sm
    +_gitstatus_daemon_p9k_:93> zsystem flock -- /private/var/folders/mn/zftkdvk150v98011k4dmb_zr0000gn/T/gitstatus.POWERLEVEL9K.501.67356.1619095118.2.lock
    +_gitstatus_daemon_p9k_:12> uname_sm=+_gitstatus_daemon_p9k_:12> uname -sm
    +_gitstatus_daemon_p9k_:12> uname_sm='darwin x86_64'
    +_gitstatus_daemon_p9k_:13> [[ 'darwin x86_64' == [^\ ]##\ [^\ ]## ]]
    +_gitstatus_daemon_p9k_:14> local uname_s=darwin
    +_gitstatus_daemon_p9k_:15> local uname_m=x86_64
    +_gitstatus_daemon_p9k_:17> [[ '' == <1-> ]]
    +_gitstatus_daemon_p9k_:20> local cpus
    +_gitstatus_daemon_p9k_:21> ((  ! 1  ))
    +_gitstatus_daemon_p9k_:21> [[ darwin == linux ]]
    +_gitstatus_daemon_p9k_:22> cpus=+_gitstatus_daemon_p9k_:22> sysctl -n hw.ncpu
    +_gitstatus_daemon_p9k_:22> cpus=8
    +_gitstatus_daemon_p9k_:27> args+=( -t 16 )
    +_gitstatus_daemon_p9k_:30> mkfifo -- /private/var/folders/mn/zftkdvk150v98011k4dmb_zr0000gn/T/gitstatus.POWERLEVEL9K.501.67356.1619095118.2.fifo
    +_gitstatus_daemon_p9k_:31> print -rnu 15 -- '               67408'
    +_gitstatus_daemon_p9k_:33> zf_rm -- /private/var/folders/mn/zftkdvk150v98011k4dmb_zr0000gn/T/gitstatus.POWERLEVEL9K.501.67356.1619095118.2.fifo
    +_gitstatus_daemon_p9k_:35> local _gitstatus_zsh_daemon _gitstatus_zsh_version _gitstatus_zsh_downloaded
    +_gitstatus_daemon_p9k_:43> local gitstatus_plugin_dir_var=_gitstatus_plugin_dir_p9k_
    +_gitstatus_daemon_p9k_:44> local gitstatus_plugin_dir=/Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus
    +_gitstatus_daemon_p9k_:45> set -- -d /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus -s darwin -m x86_64 -p 'printf '\''\001'\'' >&15' -e 15 -- _gitstatus_set_daemon_p9k_
    +_gitstatus_daemon_p9k_:47> [[ 1 == (|-|+)<1-> ]]
    +_gitstatus_daemon_p9k_:48> source /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus/install
    +/Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus/install:467> [ -z '' ']'
    +/Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus/install:468> _gitstatus_install_main /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus -d /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus -s darwin -m x86_64 -p 'printf '\''\001'\'' >&15' -e 15 -- _gitstatus_set_daemon_p9k_
    +_gitstatus_install_main:1> [ -n 5.8 ']'
    +_gitstatus_install_main:2> emulate -L sh -o no_unset
    +_gitstatus_install_main:7> local argv1=/Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus
    +_gitstatus_install_main:8> shift
    +_gitstatus_install_main:10> local no_check='' no_install='' uname_s='' uname_m='' gitstatus_dir='' dl_status='' e=''
    +_gitstatus_install_main:11> local opt='' OPTARG='' OPTIND=1
    +_gitstatus_install_main:13> getopts :s:m:d:p:e:fnh opt -d /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus -s darwin -m x86_64 -p 'printf '\''\001'\'' >&15' -e 15 -- _gitstatus_set_daemon_p9k_
    +_gitstatus_install_main:14> case d (h)
    +_gitstatus_install_main:14> case d (n)
    +_gitstatus_install_main:14> case d (f)
    +_gitstatus_install_main:14> case d (d)
    +_gitstatus_install_main:55> [ -n '' ']'
    +_gitstatus_install_main:59> [ -z /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus ']'
    +_gitstatus_install_main:63> gitstatus_dir=/Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus
    +_gitstatus_install_main:13> getopts :s:m:d:p:e:fnh opt -d /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus -s darwin -m x86_64 -p 'printf '\''\001'\'' >&15' -e 15 -- _gitstatus_set_daemon_p9k_
    +_gitstatus_install_main:14> case s (h)
    +_gitstatus_install_main:14> case s (n)
    +_gitstatus_install_main:14> case s (f)
    +_gitstatus_install_main:14> case s (d)
    +_gitstatus_install_main:14> case s (p)
    +_gitstatus_install_main:14> case s (e)
    +_gitstatus_install_main:14> case s (m)
    +_gitstatus_install_main:14> case s (s)
    +_gitstatus_install_main:99> [ -n '' ']'
    +_gitstatus_install_main:103> [ -z darwin ']'
    +_gitstatus_install_main:107> uname_s=darwin
    +_gitstatus_install_main:13> getopts :s:m:d:p:e:fnh opt -d /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus -s darwin -m x86_64 -p 'printf '\''\001'\'' >&15' -e 15 -- _gitstatus_set_daemon_p9k_
    +_gitstatus_install_main:14> case m (h)
    +_gitstatus_install_main:14> case m (n)
    +_gitstatus_install_main:14> case m (f)
    +_gitstatus_install_main:14> case m (d)
    +_gitstatus_install_main:14> case m (p)
    +_gitstatus_install_main:14> case m (e)
    +_gitstatus_install_main:14> case m (m)
    +_gitstatus_install_main:88> [ -n '' ']'
    +_gitstatus_install_main:92> [ -z x86_64 ']'
    +_gitstatus_install_main:96> uname_m=x86_64
    +_gitstatus_install_main:13> getopts :s:m:d:p:e:fnh opt -d /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus -s darwin -m x86_64 -p 'printf '\''\001'\'' >&15' -e 15 -- _gitstatus_set_daemon_p9k_
    +_gitstatus_install_main:14> case p (h)
    +_gitstatus_install_main:14> case p (n)
    +_gitstatus_install_main:14> case p (f)
    +_gitstatus_install_main:14> case p (d)
    +_gitstatus_install_main:14> case p (p)
    +_gitstatus_install_main:66> [ -n '' ']'
    +_gitstatus_install_main:70> [ -z 'printf '\''\001'\'' >&15' ']'
    +_gitstatus_install_main:74> dl_status='printf '\''\001'\'' >&15'
    +_gitstatus_install_main:13> getopts :s:m:d:p:e:fnh opt -d /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus -s darwin -m x86_64 -p 'printf '\''\001'\'' >&15' -e 15 -- _gitstatus_set_daemon_p9k_
    +_gitstatus_install_main:14> case e (h)
    +_gitstatus_install_main:14> case e (n)
    +_gitstatus_install_main:14> case e (f)
    +_gitstatus_install_main:14> case e (d)
    +_gitstatus_install_main:14> case e (p)
    +_gitstatus_install_main:14> case e (e)
    +_gitstatus_install_main:77> [ -n '' ']'
    +_gitstatus_install_main:81> [ -z 15 ']'
    +_gitstatus_install_main:85> e=15
    +_gitstatus_install_main:13> getopts :s:m:d:p:e:fnh opt -d /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus -s darwin -m x86_64 -p 'printf '\''\001'\'' >&15' -e 15 -- _gitstatus_set_daemon_p9k_
    +_gitstatus_install_main:115> shift 11
    +_gitstatus_install_main:117> : 15
    +_gitstatus_install_main:118> : /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus
    +_gitstatus_install_main:120> [ -n '' -a -n '' ']'
    +_gitstatus_install_main:125> [ -z darwin ']'
    +_gitstatus_install_main:129> [ -z x86_64 ']'
    +_gitstatus_install_main:134> local daemon=''
    +_gitstatus_install_main:135> local cache_dir=/Users/macbacupfb11/.cache/gitstatus
    +_gitstatus_install_main:137> [ -z '' ']'
    +_gitstatus_install_main:138> [ -n '' ']'
    +_gitstatus_install_main:142> [ -z '' -a -e /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus/usrbin/gitstatusd ']'
    +_gitstatus_install_main:145> [ -n '' ']'
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line=''
    +_gitstatus_install_main:163> [ -n '' ']'
    +_gitstatus_install_main:163> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line=''
    +_gitstatus_install_main:163> [ -n '' ']'
    +_gitstatus_install_main:163> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line=''
    +_gitstatus_install_main:163> [ -n '' ']'
    +_gitstatus_install_main:163> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line=''
    +_gitstatus_install_main:163> [ -n '' ']'
    +_gitstatus_install_main:163> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line=''
    +_gitstatus_install_main:163> [ -n '' ']'
    +_gitstatus_install_main:163> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line=''
    +_gitstatus_install_main:163> [ -n '' ']'
    +_gitstatus_install_main:163> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line=''
    +_gitstatus_install_main:163> [ -n '' ']'
    +_gitstatus_install_main:163> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line=''
    +_gitstatus_install_main:163> [ -n '' ']'
    +_gitstatus_install_main:163> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line=''
    +_gitstatus_install_main:163> [ -n '' ']'
    +_gitstatus_install_main:163> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line=''
    +_gitstatus_install_main:163> [ -n '' ']'
    +_gitstatus_install_main:163> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line='uname_s_glob="cygwin_nt-10.0"; uname_m_glob="i686";    file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="c96baef70b81b5a1d46adcc9e93721eaf4bdc295562bdd2baf210a6b416b9911";'
    +_gitstatus_install_main:163> [ -n 'uname_s_glob="cygwin_nt-10.0"; uname_m_glob="i686";    file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="c96baef70b81b5a1d46adcc9e93721eaf4bdc295562bdd2baf210a6b416b9911";' ']'
    +_gitstatus_install_main:165> local uname_s_glob='' uname_m_glob='' file='' version='' sha256=''
    +_gitstatus_install_main:166> eval 'uname_s_glob="cygwin_nt-10.0"; uname_m_glob="i686";    file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="c96baef70b81b5a1d46adcc9e93721eaf4bdc295562bdd2baf210a6b416b9911";'
    +_gitstatus_install_main:166> uname_s_glob=cygwin_nt-10.0
    +_gitstatus_install_main:166> uname_m_glob=i686
    +_gitstatus_install_main:166> file=gitstatusd-darwin-x86_64
    +_gitstatus_install_main:166> version=v1.3.1
    +_gitstatus_install_main:166> sha256=c96baef70b81b5a1d46adcc9e93721eaf4bdc295562bdd2baf210a6b416b9911
    +_gitstatus_install_main:168> [ -z cygwin_nt-10.0 -o -z i686 -o -z gitstatusd-darwin-x86_64 -o -z v1.3.1 -o -z c96baef70b81b5a1d46adcc9e93721eaf4bdc295562bdd2baf210a6b416b9911 ']'
    +_gitstatus_install_main:177> case darwin (cygwin_nt-10.0)
    +_gitstatus_install_main:177> case darwin (*)
    +_gitstatus_install_main:179> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line='uname_s_glob="cygwin_nt-10.0"; uname_m_glob="x86_64";  file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="18b5be49f6eb9ff1cf25e76d6f2333c7402e686e05ce5b88ca107c80504210d8";'
    +_gitstatus_install_main:163> [ -n 'uname_s_glob="cygwin_nt-10.0"; uname_m_glob="x86_64";  file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="18b5be49f6eb9ff1cf25e76d6f2333c7402e686e05ce5b88ca107c80504210d8";' ']'
    +_gitstatus_install_main:165> local uname_s_glob='' uname_m_glob='' file='' version='' sha256=''
    +_gitstatus_install_main:166> eval 'uname_s_glob="cygwin_nt-10.0"; uname_m_glob="x86_64";  file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="18b5be49f6eb9ff1cf25e76d6f2333c7402e686e05ce5b88ca107c80504210d8";'
    +_gitstatus_install_main:166> uname_s_glob=cygwin_nt-10.0
    +_gitstatus_install_main:166> uname_m_glob=x86_64
    +_gitstatus_install_main:166> file=gitstatusd-darwin-x86_64
    +_gitstatus_install_main:166> version=v1.3.1
    +_gitstatus_install_main:166> sha256=18b5be49f6eb9ff1cf25e76d6f2333c7402e686e05ce5b88ca107c80504210d8
    +_gitstatus_install_main:168> [ -z cygwin_nt-10.0 -o -z x86_64 -o -z gitstatusd-darwin-x86_64 -o -z v1.3.1 -o -z 18b5be49f6eb9ff1cf25e76d6f2333c7402e686e05ce5b88ca107c80504210d8 ']'
    +_gitstatus_install_main:177> case darwin (cygwin_nt-10.0)
    +_gitstatus_install_main:177> case darwin (*)
    +_gitstatus_install_main:179> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line='uname_s_glob="darwin";         uname_m_glob="arm64";   file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="0394e2ac880c1e190ace0346499d4670861297ecc2f84315ecb8ba3c98aa68d9";'
    +_gitstatus_install_main:163> [ -n 'uname_s_glob="darwin";         uname_m_glob="arm64";   file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="0394e2ac880c1e190ace0346499d4670861297ecc2f84315ecb8ba3c98aa68d9";' ']'
    +_gitstatus_install_main:165> local uname_s_glob='' uname_m_glob='' file='' version='' sha256=''
    +_gitstatus_install_main:166> eval 'uname_s_glob="darwin";         uname_m_glob="arm64";   file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="0394e2ac880c1e190ace0346499d4670861297ecc2f84315ecb8ba3c98aa68d9";'
    +_gitstatus_install_main:166> uname_s_glob=darwin
    +_gitstatus_install_main:166> uname_m_glob=arm64
    +_gitstatus_install_main:166> file=gitstatusd-darwin-x86_64
    +_gitstatus_install_main:166> version=v1.3.1
    +_gitstatus_install_main:166> sha256=0394e2ac880c1e190ace0346499d4670861297ecc2f84315ecb8ba3c98aa68d9
    +_gitstatus_install_main:168> [ -z darwin -o -z arm64 -o -z gitstatusd-darwin-x86_64 -o -z v1.3.1 -o -z 0394e2ac880c1e190ace0346499d4670861297ecc2f84315ecb8ba3c98aa68d9 ']'
    +_gitstatus_install_main:177> case darwin (darwin)
    +_gitstatus_install_main:181> case x86_64 (arm64)
    +_gitstatus_install_main:181> case x86_64 (*)
    +_gitstatus_install_main:183> continue
    +_gitstatus_install_main:161> IFS='' +_gitstatus_install_main:161> read -r line
    +_gitstatus_install_main:162> line='uname_s_glob="darwin";         uname_m_glob="x86_64";  file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="26d582fe9a0b2090c28e84e5e32a6d42d6988cedff51e41ec5f789512c53b0fc";'
    +_gitstatus_install_main:163> [ -n 'uname_s_glob="darwin";         uname_m_glob="x86_64";  file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="26d582fe9a0b2090c28e84e5e32a6d42d6988cedff51e41ec5f789512c53b0fc";' ']'
    +_gitstatus_install_main:165> local uname_s_glob='' uname_m_glob='' file='' version='' sha256=''
    +_gitstatus_install_main:166> eval 'uname_s_glob="darwin";         uname_m_glob="x86_64";  file="gitstatusd-${uname_s}-${uname_m}";     version="v1.3.1"; sha256="26d582fe9a0b2090c28e84e5e32a6d42d6988cedff51e41ec5f789512c53b0fc";'
    +_gitstatus_install_main:166> uname_s_glob=darwin
    +_gitstatus_install_main:166> uname_m_glob=x86_64
    +_gitstatus_install_main:166> file=gitstatusd-darwin-x86_64
    +_gitstatus_install_main:166> version=v1.3.1
    +_gitstatus_install_main:166> sha256=26d582fe9a0b2090c28e84e5e32a6d42d6988cedff51e41ec5f789512c53b0fc
    +_gitstatus_install_main:168> [ -z darwin -o -z x86_64 -o -z gitstatusd-darwin-x86_64 -o -z v1.3.1 -o -z 26d582fe9a0b2090c28e84e5e32a6d42d6988cedff51e41ec5f789512c53b0fc ']'
    +_gitstatus_install_main:177> case darwin (darwin)
    +_gitstatus_install_main:181> case x86_64 (x86_64)
    +_gitstatus_install_main:188> [ -z '' ']'
    +_gitstatus_install_main:190> local daemon=/Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus/usrbin/gitstatusd-darwin-x86_64
    +_gitstatus_install_main:191> [ ! -e /Users/macbacupfb11/.oh-my-zsh/custom/themes/powerlevel10k/gitstatus/usrbin/gitstatusd-darwin-x86_64 ']'
    +_gitstatus_install_main:192> daemon=/Users/macbacupfb11/.cache/gitstatus/gitstatusd-darwin-x86_64
    +_gitstatus_install_main:193> [ -e /Users/macbacupfb11/.cache/gitstatus/gitstatusd-darwin-x86_64 ']'
    +_gitstatus_install_main:195> [ -n /Users/macbacupfb11/.cache/gitstatus/gitstatusd-darwin-x86_64 ']'
    +_gitstatus_install_main:196> _gitstatus_install_daemon_found 0 _gitstatus_set_daemon_p9k_
    +_gitstatus_install_daemon_found:1> local installed=0
    +_gitstatus_install_daemon_found:2> shift
    +_gitstatus_install_daemon_found:3> [ 1 '=' 0 ']'
    +_gitstatus_install_daemon_found:3> _gitstatus_set_daemon_p9k_ /Users/macbacupfb11/.cache/gitstatus/gitstatusd-darwin-x86_64 v1.3.1 0
    +_gitstatus_set_daemon_p9k_:1> _gitstatus_zsh_daemon=/Users/macbacupfb11/.cache/gitstatus/gitstatusd-darwin-x86_64
    +_gitstatus_set_daemon_p9k_:2> _gitstatus_zsh_version=v1.3.1
    +_gitstatus_set_daemon_p9k_:3> _gitstatus_zsh_downloaded=0
    +_gitstatus_install_main:197> return
    +_gitstatus_daemon_p9k_:49> [[ -n /Users/macbacupfb11/.cache/gitstatus/gitstatusd-darwin-x86_64 ]]
    +_gitstatus_daemon_p9k_:50> [[ -n v1.3.1 ]]
    +_gitstatus_daemon_p9k_:51> [[ 0 == [01] ]]
    +_gitstatus_daemon_p9k_:53> ((  UID == EUID  ))
    +_gitstatus_daemon_p9k_:54> local home=/Users/macbacupfb11
    +_gitstatus_daemon_p9k_:62> [[ -x /Users/macbacupfb11/.cache/gitstatus/gitstatusd-darwin-x86_64 ]]
    +_gitstatus_daemon_p9k_:63> HOME=/Users/macbacupfb11 /Users/macbacupfb11/.cache/gitstatus/gitstatusd-darwin-x86_64 -G v1.3.1 -s -1 -u -1 -d -1 -c -1 -m -1 -v DEBUG -t 16

  System information:

    zsh:      5.8
    uname -a: Darwin Edwards-MacBook-Pro-2.local 20.3.0 Darwin Kernel Version 20.3.0: Thu Jan 21 00:07:06 PST 2021; root:xnu-7195.81.3~1/RELEASE_X86_64 x86_64

  If you need help, open an issue and attach this whole error message to it:

    https://github.com/romkatv/gitstatus/issues/new
    ~ ················································ at 05:38:43 AM  ─╮
❯

**Comment by mcornella** [comment, marked as answer]
This is a gitstatus issue. The error tells you where to find support for it:


> If you need help, open an issue and attach this whole error message to it:
> 
> ```
> https://github.com/romkatv/gitstatus/issues/new
> ```
